config.ini

```diff
@@ -35,6 +35,19 @@ session.secure = False
 # wsgi server configuration
 ###
 
+# This configures Pyramid/Waitress to respect X-Forwarded-* headers which we
```

Comment:
I'm curious what pypicloud needs or wants the X-Forwarded headers for. It's not for routing, right? Isn't it basically so we know the client IP?